### PR TITLE
PEP 609: Rename to Python Packaging Authority (PyPA) Governance

### PR DESCRIPTION
**Pull request by pradyunsg**
This makes it clearer what the acronym stands for, by presenting the
full form in addition to the acronym.

### CODE REVIEW

File: pep-0609.rst
Comment (AA-Turner) [inline pep-0609.rst:2]: The acronym is introduced in the first sentence of the text, this shortens the title by seven characters:
```suggestion
Title: Python Packaging Authority Governance
```

A

**Comment by AA-Turner** [comment]
On a technical reading of PEP-609, a committer vote is required for *any* change -- @pradyunsg we could add the do not merge label to this PR if such a vote is to take place. (Pragmatically it seems overkill, but not my decision to make!).

A

----

> PyPA committer votes are required for, and limited to, the following kinds of proposals:
>
> ...
>
> **Updates to the Governance/Specification Processes**
> 
> Proposing changes to how the PyPA operates, including but not limited to changes to its specification and governance processes, and this PEP.

File: pep-0609.rst
Comment (pradyunsg) [reply pep-0609.rst:2]: I'd like to keep the acronym, for folks who don't click into the PEP, on the Packaging topic index.

**Comment by pradyunsg** [comment]
> it seems a committer vote is required for any change to the PEP

No, it's needed for process changes. I don't think this needs a PyPA committer vote -- this PR changes nothing about PyPA's processes.

**Review by AA-Turner — APPROVED** [review]

**Comment by AA-Turner** [comment]
@pradyunsg is this OK to be merged?

A

**Comment by pradyunsg** [comment]
I believe so, yes.

**Pull request merged**
